Gruntfile.js

```diff
@@ -162,7 +180,7 @@ module.exports = function(grunt) {
 
 	require('matchdep').filterDev('grunt-*').forEach( grunt.loadNpmTasks );
 
-	grunt.registerTask('default', ['watch']);
+	grunt.registerTask('default', ['sass','autoprefixer','cssjanus','cssmin','jshint','uglify']);
```

Comment:
@fjarrett @schrapel I've changed the default grunt command to only compile and run lint (this is what we used as default for out other plugins out there. `grunt watch` if you want to start watching all those files.